ui/app/AppLayouts/Chat/ChatColumn/ChatMessages.qml

```diff
@@ -118,16 +118,16 @@ SplitView {
                     radius: 16
                 }
                 onClicked: {
-                    root.newMessages = 0
+                    svRoot.newMessages = 0
```

Comment:
I fixed these in my today's PR...

Reply:
OH! Then we can ignore this one :D

Reply:
No! mine is not merged, so if yours comes first, then I will discard mine.

Reply:
Actually let me cancel mine right now

Reply:
Closed https://github.com/status-im/status-desktop/pull/2860